packages/apollo-engine-reporting/src/__tests__/extension.test.ts

```diff
@@ -65,11 +65,15 @@ test('trace construction', async () => {
     addTrace,
     'schema-hash',
   );
+  // XXX This test will not be a true mirror of how Apollo Server will work
```

Comment:
I don't know what this means.

Reply:
The extension relies on `operationName` being set in the requestPipeline, for instance.